p2p_wallet/Scenes/Main/Main.ViewController.swift

```diff
@@ -93,6 +94,7 @@ extension Main {
             let lockView = LockView()
             UIApplication.shared.windows.last?.addSubview(lockView)
             lockView.autoPinEdgesToSuperviewEdges()
+            solanaTracker.stopTracking()
```

Comment:
it is a bit unclear why here is stop and start is in different place. why is it so?